extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1161,3 +1343,12 @@ function getFirstInstanceValue_(instances, extractFn) {
   }
   return null;
 }
+
+/**
+ * Used to manage messages for different fluid ad slots.
+ *
+ * Maps a sentinel value to an object consisting of the impl to which that
+ * sentinel value belongs and the corresponding message handler for that impl.
+ * @type{!Object<string, !Object>}
+ */
+const fluidListeners = {};
```

Comment:
Can we put this at the top of the file?  You can also use the typedef here:

Object<string, !{{instance: !AmpAdNetworkDoubleclickImpl, connectionEstablished: boolean}}>

Reply:
Done.